websites/J/JAVX/dist/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.3",
+  "author": {
+    "name": "Minx#9400",
+    "id": "829346671544107088"
+  },
+  "service": "JAVX",
+  "description": {
+    "en": "JAVX - Watch JAV, Watch Porn, Watch Sex Video 18+ Full HD Uncensored",
+    "vi_VN": "JAVX - Xem JAV, Xem Porn, Xem Phim Sex 18+ Full HD Không Che Mới Nhất"
+  },
+  "url": "javx.net",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/0Jwz0Ct.png",
+  "thumbnail": "https://i.imgur.com/9lfQfm2.png",
```

Comment:
I would like you to change images to something different than blur please.. and thumbnail should show main page.